api/helpers/gamestate/moves/resolve/execute.js

```diff
@@ -75,6 +66,19 @@ module.exports = {
       default:
         return exits.error(new Error(`${oneOff.rank} is not a valid one-off rank`));
     }
+    
+    result = {
```

Comment:
Some of the data that's being set between lines 70-81 is the same as what needs setting  in the coalesced cases for 3, 4, 5, and 7. Let's split up the assignment to result into three chunks. The general idea is that some data needs setting for every one-off, and some data is specific to the one-offs that require extra user input (e.g. picking a card from the scrap with a three), and then some data gets set only for the single-stage one-offs e.g. ace and six which compeltely finish resolving in this helper e.g. incrementing the turn happens for aces/sixes etc but not for threes/fours/fives/sevens because those need to wait for additional user input in a subsequent request. We should split into 3 times we assign to `result`:

1. One up above back where it was that has just the shared data
2. Another in the cases for 3, 4, 5, and 7 that sets their specific data _and then returns `exits.success(result)`_
3. Another block below, where you have it here, which just sets the data that is shared by the one-offs like ace which finish resolving in a single step.

The first section (where we previously had it) should  remove the line above where we scrap the twos `result.scrap.push(result.oneOff);` (that will be moved below to the third section, and then it should set

```
result = {
      ...result,
      ...requestedMove,
      playedBy,
      resolved: oneOff,
      twos: [],
    };
}
```
because this is the data that's common to all one-offs.

The section, in the cases for 3, 4, 5, and 7 should just set `result.phase = oneOff.rank` and then `return exits.success(result)` because it only sets the phase and waits for the next move to scrap the oneOff and actually do its effect.

The third section is just for the single stage one-offs that fully resolve and it should additionally set:
```
result.scrap.push(result.oneOff);
result = {
  ...result,
  oneOff: null,
  oneOffTarget: null,
  oneOffTargetType: null,
  turn: result.turn + 1,
};
```

since this is the data for when the turn ends and a one-off finishes resolving